.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,48 @@
+name: 🐛 Bug Report
+description: Report a bug encountered while using Cluster API
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What steps did you take and what happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please follow the guidelines described in https://github.com/kubernetes-sigs/cluster-api/blob/main/SECURITY_CONTACTS
+      placeholder: "A clear and concise description on how to REPRODUCE the bug."
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: capiVersion
+    attributes:
+      label: Cluster-api version
```

```suggestion
      label: Cluster API version
```
Given the previous finding from Vince. Did you maybe forgot to push some changes / lose some changes?
(please also adjust Cluster API in other places)

---

Thanks for catching, looks like I messed up the previous commit, fixed it now.